dwbb/src/interface/DW_asymfifoctl_2c_df.scala

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW_asymfifoctl_2c_df
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import chisel3.util.log2Ceil
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(
+    dataSWidth: Int = 16,
+    dataDWidth: Int = 8,
+    ramDepth: Int = 8,
+    memMode: Int = 3,
+    archType: String = "pipeline",
+    fSyncType: String = "pos_pos_sync",
+    rSyncType: String = "pos_pos_sync",
+    byteOrder: Boolean = false,
+    flushValue: Boolean = false,
+    clkRatio: Int = 1,
+    ramReExt: Boolean = false,
+    errMode: Boolean = false,
```

Comment:
should not be `Boolean`? please make sure?

Reply:
Using `String`s instead of `Boolean`s in the revised code.